images/php/Dockerfile

```diff
@@ -9,4 +9,6 @@ RUN chmod 777 /tmp
 ENV PHP_VERSION=82
 ENV COMPOSER_ALLOW_SUPERUSER=1
 RUN /tmp/install-php.sh && \
-    rm /tmp/install-php.sh
+    rm /tmp/install-php.sh && \
+    echo "COMPOSER_ALLOW_SUPERUSER=$COMPOSER_ALLOW_SUPERUSER" >> /etc/profile.d/php.sh && \
```

Comment:
Nit: Would be a bit nicer if we do everything installation related in /tmp/install-php.sh